File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    if(arr1.length!==arr2.length){
        return arr1.length>arr2.length?1:-1
    }
    else{
        let a1 = arr1.reduce((acc,curr) => acc+curr, 0);
        let a2 = arr2.reduce((acc,curr) => acc+curr, 0);
        return a1>a2?1:a2>a1?-1:0
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
import java.util.*;
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        if(arr1.length==arr2.length){
            int sum1 = Arrays.stream(arr1).sum();
            int sum2 = Arrays.stream(arr2).sum();
            
            return sum1==sum2?0:sum1>sum2?1:-1;
        }
        else{
            return arr1.length>arr2.length?1:-1;
        }
    }
}